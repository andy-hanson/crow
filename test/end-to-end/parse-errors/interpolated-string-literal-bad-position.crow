main void()
	info log "{x "y"}"
