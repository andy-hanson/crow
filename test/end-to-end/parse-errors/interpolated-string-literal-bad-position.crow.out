exit code: 1
stderr:
test/end-to-end/parse-errors/interpolated-string-literal-bad-position.crow 2:18-2:19 Expected '}'.
